tests/61push/09_notifications_api.pl

```diff
@@ -30,41 +30,45 @@
             body => "Test message 2",
          );
       })->then( sub {
-         do_request_json_for( $user1,
-            method  => "GET",
-            uri     => "/unstable/notifications",
-         );
-      })->then( sub {
-         my ( $body ) = @_;
+         retry_until_success {
+            do_request_json_for( $user1,
+               method  => "GET",
+               uri     => "/unstable/notifications",
```

Comment:
we should really change these to be /r0/, though leaving them for another PR makes some sense